curriculum/challenges/english/07-scientific-computing-with-python/learn-interfaces-by-building-an-equation-solver/662fbcef5f05e1b84f541a0c.md

```diff
@@ -0,0 +1,75 @@
+---
+id: 662fbcef5f05e1b84f541a0c
+title: Step 13
+challengeType: 20
+dashedName: step-13
+---
+
+# --description--
+
+Each equation object will be instantiated passing as many arguments as the coefficients of the equation, starting from n-th degree of \\( x \\) down to the zero-th degree, including the possible coefficient with the value of `0`.
+
```

Comment:
Maybe we can add something like this to further show what the instruction means:

```md
For example, (4,5) would translate to \\( 4x^1 + 5x^0 = 0 \\) or  \\( 4x + 5 = 0 \\) with `4` being the coefficient of the first(highest here) degree and `5` the coefficient of the zero-th degree.
```